README.md

```diff
@@ -92,6 +92,8 @@ with ZoomClient('API_KEY', 'API_SECRET') as client:
 * client.user.pending(...)
 * client.user.get(...)
 * client.user.get_by_email(...)
+* client.user.get_settings(...)
+* client.user.update_settings(...)
```

Comment:
Thank you for adding these new methods to the README! We're in the process of switching over to Sphinx for our documentation (#183), so can I trouble you to add this to `/docs/source/index.rst` as well so that we don't forget to add it?